simple.ml

```diff
@@ -0,0 +1,26 @@
+let tbl = Hashtbl.create 1000000 in
```

Comment:
Might be "cheating" but we know the table has 32187 entries so make it ~33k?

Reply:
Might as well!